### DISCUSSION TITLE
Hello there.

### DISCUSSION THREAD

**Discussion by kovitikus** [evennia/evennia#2281] [Greetings & Introductions]
Introducing myself isn't something I normally do in communities, but I'm very active around Evennia, so I thought it might help.

I'm working on a dark action-combat MUD called [Hecate](https://github.com/kovitikus/hecate). Many folk seem to enjoy reading and using the code I've written to help them in their own projects. Feel free to "steal" anything you see on my project. Hecate is licensed as public domain!

I also sometimes write things in the Hecate [wiki](https://github.com/kovitikus/hecate/wiki), including tutorials for Evennia!

I'm a noob to Python and Evennia, but I learned that with enough reading of the Evennia documentation, hours of googling, and sheer trial and error that I've picked up on the concept pretty quickly. So far, it's been the most productive attempt I've ever made at learning to code or develop a game, on any engine. @Griatch has done a great job at maintaining and explaining how the engine works.

The Evennia [Discord](https://discord.gg/NecFePw) and [IRC](http://webchat.freenode.net/?channels=evennia&uio=MT1mYWxzZSY5PXRydWUmMTE9MTk1JjEyPXRydWUbb) channels have blessed me with much wisdom. If you are new around here, I highly suggest engaging there as well. Discussions here on the wiki are new, but I could see them being a great boon for learning soon enough!

**Comment by Griatch** [comment]
:wave: Fun to see people starting to use these discussion-forums. You are doing great progress from the sounds of it here and in chat!
